tests/unit/nupic/algorithms/sdr_classifier_test.py

```diff
@@ -0,0 +1,445 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2016, Numenta, Inc.  Unless you have an agreement
+# with Numenta, Inc., for a separate license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+"""Unit tests for SDRClassifier module."""
+
+CL_VERBOSITY = 0
+
+import cPickle as pickle
+import types
+import unittest2 as unittest
+
+import numpy
+import tempfile
+
+from nupic.algorithms.sdr_classifier import SDRClassifier
+
+class SDRClassifierTest(unittest.TestCase):
+  """Unit tests for SDRClassifier class."""
+
+
+  def setUp(self):
+    self._classifier = SDRClassifier
+
+
+  def testInitialization(self):
+    c = self._classifier([1], 0.1, 0.1, 0)
+    self.assertEqual(type(c), self._classifier)
```

Comment:
We can add some tests of constructors with bad parameters - they should throw exceptions.